src/components/Snackbar/index.js

```diff
@@ -19,9 +19,17 @@ const Alert = ({ iconMapping = defaultAlertIconMapping, ...otherProps }) => {
   return <MuiAlert elevation={6} variant="filled" iconMapping={iconMapping} {...otherProps} />;
 };
 
-const SimpleSnackbar = ({ open, text, severity, duration, onClose, alertStyle, ...otherProps }) => {
+const SimpleSnackbar = ({
+  open,
+  text,
+  severity,
+  duration = 10000,
+  onClose,
+  alertStyle,
+  ...otherProps
+}) => {
   return (
-    <Snackbar open={open} autoHideDuration={duration || 10000} onClose={onClose} {...otherProps}>
+    <Snackbar open={open} autoHideDuration={duration} onClose={onClose} {...otherProps}>
       <Alert style={alertStyle || { textAlign: 'center' }} onClose={onClose} severity={severity}>
         {text}
       </Alert>
```

Comment:
Since the inside does nothing other than return, you could also use:
```
}) => (
  <Snackbar open={open} autoHideDuration={duration} onClose={onClose} {...otherProps}>
    <Alert style={alertStyle || { textAlign: 'center' }} onClose={onClose} severity={severity}>
      {text}
    </Alert>
  </Snackbar>
);
```